### PR TITLE
Translate Personal Profile mockup to code

### PR DESCRIPTION
**Pull request by mrcongliu**
https://trello.com/c/p9CkS5ra/174-translate-personal-profile-mockup-to-code

### CODE REVIEW

**Comment by mrcongliu** [comment]
I manually removed all the changes I made last time(change localhost).
This commit should only contain Personal Profile mockup now.

File: src/components/UI/XButton/XButton.module.css [hunk 1]
```diff
@@ -20,9 +20,11 @@
 }
 
 .DisplayInterests {
-    position: absolute;
-    right: 2px;
-    top: 2px;
-    font-size: 20px;
-    color: #6831de;
+    font-size: 16px;
```
Comment (ArnasDickus) [inline src/components/UI/XButton/XButton.module.css:31]: In your mockup. I didn't see displayInterests. Why did you change css here?

File: src/pages/PersonalProfile/PersonalProfile.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.profile-page {
+  padding: 0 2rem;
+}
+
+.profile-title {
+  color: #6831de;
+}
```
Comment (ArnasDickus) [inline src/pages/PersonalProfile/PersonalProfile.css]: This needs formating.  Try running npm run format

File: src/pages/PersonalProfile/ProfileContent.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.text-interests {
+  font-size: 22px;
+  font-weight: bold;
+  color: #6831de;
+}
```
Comment (ArnasDickus) [inline src/pages/PersonalProfile/ProfileContent.css]: Needs formating

File: src/pages/PersonalProfile/ProfileCard.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from '../../components/UI/Button/Button';
+import styled from 'styled-components';
+import './ProfileCard.css';
+
+const Card = styled.div`
+    width: 20%;
```
Comment (ArnasDickus) [inline src/pages/PersonalProfile/ProfileCard.js]: What these 2 rules do?
`width: 20%;
 height: 90%;`

File: src/pages/PersonalProfile/PersonalProfile.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import './PersonalProfile.css';
+import ProfileCard from './ProfileCard';
+import ProfileContent from './ProfileContent';
+import USER_DATA from './USER_DATA';
+
+const Profile = styled.div`
+    display: flex;
+    width: 80vw;
```
Comment (ArnasDickus) [inline src/pages/PersonalProfile/PersonalProfile.js]: This ` width: 80vw;` css property, not good with responsiveness
width-rule

File: src/pages/PersonalProfile/ProfileCard.js (hunk 4 above)
Comment (ArnasDickus) [reply src/pages/PersonalProfile/ProfileCard.js]: width: 20% is messing up with responsiveness.
width-rule-problem

File: src/pages/PersonalProfile/ProfileCard.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    height: 90%;
+    text-align: center;
+    margin: 2rem 2.5rem;
+    @media (max-width: 720px) {
+        width: 50%;
```
Comment (ArnasDickus) [inline src/pages/PersonalProfile/ProfileCard.js]: This rule is messing with css. It should be centered.
width-rule-problem-2

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for PR.

My concerns:
1) In mockup, about me, Interests, name, join Date, current location are not bold.
2) Between About me, and interests it shouldn't be such gap. It should be close to text like in the mockup.
3)  You need to improve mobile responsiveness. From what I saw it's mostly a problem with width and a few unnecessary height properties.

File: src/components/UI/XButton/XButton.module.css (hunk 1 above)
Comment (mrcongliu) [reply src/components/UI/XButton/XButton.module.css:31]: Hi Arnas, inside interest section, there are three Xbuttons that contains tags for each interest. Aren't they suppose to be the DisplayInterests?

Comment (ArnasDickus) [reply src/components/UI/XButton/XButton.module.css:31]: Yes, you right on that one.

**Review by ArnasDickus — APPROVED** [review]

File: public/index.html [hunk 7]
```diff
@@ -27,6 +27,7 @@
             href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.54.0/mapbox-gl.css"
             rel="stylesheet"
         />
+        <script src="https://kit.fontawesome.com/b63ba79241.js"></script>
```
Comment (ArnasDickus) [inline public/index.html:30]: I'm curious what does this js do?

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
So you fixed almost everything. The only thing left is to add wrapper for text like in [mockup](https://trello.com/c/9G76INBZ/121-personal-profile-page-seen-by-others) so that text won't be so close to the navbar.

 
friends-profile

**Review by ArnasDickus — APPROVED** [review]

**Comment by ArnasDickus** [comment]
Alright, I have no more concerns. Good work.
